website/content/docs/audit/index.mdx

```diff
@@ -89,6 +89,57 @@ using the [secrets enable](/vault/docs/commands/secrets/enable) command, and the
 
 [auth tune](/vault/docs/commands/auth/tune)
 
+## Audit request headers
+
+Vault Operators can audit specific HTTP headers when those headers are present in
+the incoming client request. Configuration for audit request headers is applied via
+the [Vault API](/vault/api-docs/system/config-auditing).
+
+When Vault Operators [create](/vault/api-docs/system/config-auditing#create-update-audit-request-header)
+an exception allowing a request header to appear in the audit log, by default the
+value of the header is **not** HMAC'd, this is [configurable](/vault/api-docs/system/config-auditing#hmac)
+per header.
+
+### Default headers
+
+<Warning title="Behavior changes">
+
+  Prior to Vault 1.17 **only** headers configured by the Vault Operator could appear
+  in an audit log.
+
+  As of Vault 1.17 this behavior changes, see below for further information.
+
+</Warning>
```

Comment:
```suggestion
```
The core of the warning is "keep reading", so this isn't a useful aside.

Reply:
What do you think would be a good way to draw the reader's attention on the page to let them know: something in Vault isn't the same now which impacts what can appear in raw text in your audit log?

Reply:
TBH, I'm not sure a warning here is appropriate. We should be documenting the change in the upgrade guide. If folks are upgrading w/o bothering to read *that* doc, that's not something we can control =\